Expensify/App
# Prevent sleep timer from running on native platforms

### Details
This sleep timer thing is wreaking certain havoc on native iOS and Android. Not sure why, but probably it should be disabled for now to restore performance on native apps. My current theory is that other work on the JS thread can cause the timer to get out of whack and cause reconnection callbacks to fire.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/157445

### Tests
**iOS/Android**
1. Test navigating around the app
2. Verify that the reconnection callbacks are not firing randomly
3. Verify that putting the app into the background and bringing it active again does fire the reconnection callbacks

**Web/Desktop**
1. Test navigating around the app
2. Close your laptop screen and reopen it
3. Verify the clock timer works for this scenario
4. Verify that minimizing the browser or app window and maximizing it again does not cause the reconnection callbacks to fire 

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
